lib/library.rb

```diff
@@ -0,0 +1,68 @@
+require 'date'
+require 'yaml'
+
+class Library
+  attr_accessor :items
+
+  def initialize
+    @items = YAML.load_file('./lib/data.yml') || []
+  end
+
+  def check_out(item, user)
+  case
+    when book_shelf_okay?(user) then perform_check_out(item, user)
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)